baseline.yaml

```diff
@@ -649,6 +649,26 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-13
+    maturity_level: 1
+    category: Documentation
+    criteria: |
+       The project documentation MUST include a 
+       descriptive statement about how the project 
+       is supported/maintained and what is not supported. 
+    objective: |
+       Identify and set expectations for downstream 
+       consumers around what updates the project may
+       or may not provide and when no further updates
+       will be provided.
+    implementation: |
+       Create a status check that checks the project's
+       version control system for support and/or lifecycle
+       statements or machine-readable metadata.
```

Comment:
I still think we should explicitly allow for other methods, but so long as the implementation is a "Pirates's Code", then I won't block on this.

Reply:
maybe instead of "VCS" we sub something like "publicly-available collateral"?

Reply:
That is an acceptably vague compromise to me.